plugins/main/public/styles/inventory.scss

```diff
@@ -13,10 +13,6 @@
     border-top: 1px solid #d3dae6;
   }
 
-  .details-row {
-    background: #fcfdfe;
-  }
```

Comment:
**question**: why is this style removed?

Reply:
This style is removed because otherwise a grey backgroung is added from above the search to below the callout.

<img width="1789" alt="image" src="https://github.com/user-attachments/assets/9a469411-e028-40c9-8022-d900efcaac78">

Reply:
Fixed in commit [0ee0ee4](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6868/commits/0ee0ee402d76898fb14b27318a912b566c2eba98)